roles/kubernetes/preinstall/tasks/0081-ntp-configurations.yml

```diff
@@ -74,8 +74,33 @@
     - not is_fedora_coreos
     - not ansible_os_family in ["Flatcar", "Flatcar Container Linux by Kinvolk"]
 
-- name: Set timezone
+- name: Gather selinux facts
+  ansible.builtin.setup:
+    gather_subset: selinux
+  when:
+    - ntp_timezone
+    - ansible_os_family == "RedHat"
+
+- name: Put SELinux in permissive mode, logging actions that would be blocked.
+  when:
```

Comment:
Thanks @bbaassssiiee 

In other parts of Kubespray, when is usually placed ​​below​​ the action (task). It's recommended to ​​keep it consistent​​.